conformance-test/libraryMethods.ts

```diff
@@ -0,0 +1,396 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {O_APPEND} from 'constants';
+import {file} from 'tmp';
+import {Bucket, File, HmacKey, Iam, Notification, Storage} from '../src';
+const fs = require('fs');
```

Comment:
Nitpicky but can this be changed to `import * as from 'fs';` just so there isn't mixed imports and requires.